Tests/FluentAssertions.Specs/Collections/GenericCollectionAssertionsSpecs.cs

```diff
@@ -1931,5 +1931,237 @@ public void When_inspectors_count_does_not_equal_asserting_collection_length_it_
             act.Should().Throw<XunitException>().WithMessage("*because we want to test the failure*");
         }
         #endregion
+
+        #region Satisfy (Predicates)
+
+        [Fact]
+        public void When_collection_element_at_each_position_matches_predicate_at_same_position_Satisfy_should_not_throw()
```

Comment:
🙃 Normally we try to avoid using the literal name of an assertion method in the test name so to keep it functional, but for those new tests, I'm struggling myself to keep them clear enough.

Reply:
Yeah. I had the same challenge :) There are several popular test naming conventions https://dzone.com/articles/7-popular-unit-test-naming. I think that the ones having the test method in the name work best for the GenericCollectionAssertionsSpecs class which tests several different APIs. Another option could be to have separate class per API under test.
Anyways, if you have a suggestion how to change this - I will be happy to implement.

Reply:
> There are several popular test naming conventions https://dzone.com/articles/7-popular-unit-test-naming

Indeed. We've been using 6, but I've also used 5 on a client code base. I definitely stay away from 1 and 2. 

> Another option could be to have separate class per API under test.

This is what I've been thinking about as well.

Reply:
I moved the Satisfy tests to a separate class